src/cljs/athens/events/remote.cljs

```diff
@@ -0,0 +1,458 @@
+(ns athens.events.remote
+  "`re-frame` events related to `:remote/*`."
+  (:require
+    [athens.common-events        :as common-events]
+    [athens.common-events.schema :as schema]
+    [athens.db                   :as db]
+    [malli.core                  :as m]
+    [malli.error                 :as me]
+    [re-frame.core               :as rf]))
+
+
+;; Connection Management
+
+(rf/reg-event-fx
+  :remote/connect!
+  (fn [{:keys [db]} [_ connection-config]]
+    (js/console.log ":remote/connect!" (pr-str connection-config))
+    {:db                     (-> db
+                                 (dissoc :db/filepath)
+                                 (assoc :db/remote connection-config))
+     :remote/client-connect! connection-config
+     :local-storage/set!     ["db/remote" connection-config]
```

Comment:
@neotyk Connection config is a map so why are we not serializing it while saving in local storage?

Reply:
Because didn't implement config, yet